modules/servodyn/src/ServoDyn.f90

```diff
@@ -939,12 +950,15 @@ SUBROUTINE SrvD_CalcOutput( t, u, p, x, xd, z, OtherState, y, m, ErrStat, ErrMsg
             CALL SetErrStat( ErrStat2, ErrMsg2, ErrStat, ErrMsg, RoutineName )
       END IF
       
+      ! Set ServoDyn output for flap deflection angle
+      y%BlFlapCom(1:p%NumBl) = m%dll_data%BlFlapCom
```

Comment:
This will give you a seg fault if the turbine doesn't have 3 blades. Use `y%BlFlapCom(1:p%NumBl) = m%dll_data%BlFlapCom(1:p%NumBl)` instead

Reply:
Also, if ServoDyn is not using the DLL for control, you will be using uninitialized data here. Either put an `if` statement around setting `y%BlFlapCom`, or initialize m%dll_data%BlFlapCom to 0.

Reply:
Good point.  I'm adding your changes.